src/discrete-layer/views/discrete-layer-view.tsx

```diff
@@ -140,7 +141,10 @@ const DiscreteLayerView: React.FC = observer(() => {
                   <DrawerSubtitle>Subtitle</DrawerSubtitle>
                 </DrawerHeader>
                 <DrawerContent>
-                  <div style={{backgroundColor: 'red', height: '100%', width:'100%'}}></div>
+                  <LayersResultsComponent 
+                    style={{height: '450px',width: '100%'}}
+                  />
+                  {/* <div style={{backgroundColor: 'red', height: '100%', width:'100%'}}></div> */}
```

Comment:
why commented?

Reply:
It's still no production, helps in dev time, removed anyway